core/proofs/src/elgamal.rs

```diff
@@ -192,7 +192,11 @@ mod tests {
         let params = &JubjubBls12::new();
         let p_g = FixedGenerators::NoteCommitmentRandomness; // 1
         let rng = &mut XorShiftRng::from_seed([0xbc4f6d44, 0xd62f276c, 0xb963afd0, 0x5455863d]);
+<<<<<<< HEAD
```

Comment:
conflicted

Reply:
yeah I modify it locally, so I'll push it.

Reply:
OMG my code pass the test!